umbra/umbra_64.c

```diff
@@ -1261,6 +1262,34 @@ umbra_insert_app_to_shadow_arch(void *drcontext,
                                 reg_id_t *scratch_regs,
                                 int num_scratch_regs)
 {
+#if defined(AARCH64)
+    ASSERT(num_scratch_regs > umbra_num_scratch_regs_for_translation_arch(),
+        "umbra_insert_app_to_shadow_arch requires at least one scracth register provided");
+
+    reg_id_t tmp = *scratch_regs;
+
+    instru_insert_mov_pc(drcontext, ilist, where, opnd_create_reg(tmp),
+                             OPND_CREATE_INT64(map->mask));
```

Comment:
Please fix indentation; the second line needs to be under drcontext.

Reply:
done